test/constructors.jl

```diff
@@ -65,5 +65,16 @@ module TestConstructors
     @test isequal(NullableArray{Bool, 2}(),
                   NullableArray{Bool, 2}(Array(Bool, 0, 0)))
 
-
+    # test conversion from arrays of nullables
+    array_of_nulls = Nullable{Int}[Nullable(1), Nullable(2), Nullable(3), Nullable()]
+    @test isa(NullableArray(array_of_nulls), NullableArray{Int,1})
+    @test isequal(NullableArray(array_of_nulls), NullableArray{Int,1}([1,2,3,4],[false,false,false,true]))
```

Comment:
Use `===` rather than isequal: that way, you'll check the actual type, not the `isequal` method. That will allow you to remove the lines using `isa`.